live-ingester/src/main/java/org/alfresco/hxi_connector/live_ingester/adapters/messaging/repository/filter/RepoEventFilterApplier.java

```diff
@@ -27,11 +27,14 @@
 package org.alfresco.hxi_connector.live_ingester.adapters.messaging.repository.filter;
 
 import org.alfresco.hxi_connector.live_ingester.adapters.config.properties.Filter;
-import org.alfresco.repo.event.v1.model.DataAttributes;
 import org.alfresco.repo.event.v1.model.NodeResource;
-import org.alfresco.repo.event.v1.model.RepoEvent;
 
 public interface RepoEventFilterApplier
 {
-    boolean applyFilter(RepoEvent<DataAttributes<NodeResource>> repoEvent, Filter filter);
+    boolean allowNode(NodeResource nodeResource, Filter filter);
+
+    default boolean allowNodeBefore(boolean currentlyAllowed, NodeResource nodeResource, Filter filter)
```

Comment:
I think it would be worth adding a comment to say when this default implementation is appropriate:
```
/**
 * Default implementation can be used to filter based on properties that are always present in the "before" resource. It
 * is not suitable if the property is omitted when unchanged.
 */
```

Reply:
Also we could change the parameter name to `NodeResource nodeResourceBefore` (as you've done in the overridden method).
```

Reply:
I renamed the method to `isNodeBeforeAllowe` to follow naming convention of other renamed method in this interface and renamed the paramters as suggested.
Also, I added javadocs to both methods/